src/modules/inventory/components/ManagerShowProducts/AllProducts.tsx

```diff
@@ -130,16 +105,14 @@ const AllProducts = () => {
           Add Component
         </Button>
       </div>
-      <DataGrid
-        rows={allRows}
-        columns={columns}
-        initialState={{
-          pagination: {
-            paginationModel: { page: 0, pageSize: 5 },
-          },
-        }}
-        pageSizeOptions={[5, 10]}
-        checkboxSelection
+      <TableComponent
+        dataObject={dataObject}
+        tableSize='large'
+        showDeleteButton={true}
+        showEditButton={true}
```

Comment:
אפשר לכתוב סתם showDeleteButton showEditButton, ערך ברירת המחדל הוא true
https://stackoverflow.com/questions/38659884/using-boolean-value-of-attributes-in-jsx